content/en/serverless/aws_lambda/opentelemetry.md

```diff
@@ -0,0 +1,186 @@
+---
+title: AWS Lambda and OpenTelemetry
+kind: documentation
+further_reading:
+  - link: '/opentelemetry/'
+    tag: 'Documentation'
+    text: 'OpenTelemetry in Datadog'
+---
+
+[OpenTelemetry][1] is an open source observability framework that provides IT teams with standardized protocols and tools for collecting and routing telemetry data.
+
+This page discusses using OpenTelemetry with Datadog Serverless Monitoring for AWS Lambda. For more information, including how to use OpenTelemetry in non-serverless environments, see [OpenTelemetry in Datadog][2].
+
+## Instrument AWS Lambda with OpenTelemetry
+
+There are multiple ways to instrument AWS Lambda functions with OpenTelemetry and send the data to Datadog:
+
+- [OpenTelemetry API support within the Datadog tracers](#opentelemetry-api-support-within-datadog-tracers) (recommended)
+- [OpenTelemetry SDK within the Datadog Agent](#opentelemetry-sdk-within-the-datadog-agent) (beta)
+
+### OpenTelemetry API support within Datadog tracers
+
+The Datadog tracing library, which is included in the Datadog Lambda Extension upon installation, accepts the spans and traces generated by OpenTelemetry-instrumented code, processes the telemetry, and sends it to Datadog.
+
+You can use this approach if, for example, your code has already been instrumented with the OpenTelemetry API. You may also use this approach if you want to instrument using vendor-agnostic code with the OpenTelemetry API while still gaining the benefits of using the Datadog tracing libraries.
+
+To instrument AWS Lambda with the OpenTelemetry API, set the environment variable `DD_TRACE_OTEL_ENABLED` to `true`.
+
+See [Custom instrumentation with the OpenTelemetry API][3] for more details.
+
+### OpenTelemetry SDK within the Datadog Agent
+
+<div class="alert alert-warning">This feature is in beta.</div>
+
+This approach is analogous to [OLTP Ingest in the Datadog Agent][4].
+
+1. Tell OpenTelemetry to export spans to the Datadog Lambda Extension. Then, add OpenTelemetry's instrumentation for AWS Lambda. This is analogous to adding the tracing layer.
```

Comment:
Customers may not know what "this is analagous to the tracing layer" means. Should we just remove it?